ddtrace/contrib/internal/azure_eventhubs/patch.py

```diff
@@ -0,0 +1,204 @@
+import os
+from typing import Dict
+
+import azure.eventhub as azure_eventhub
+import azure.eventhub.aio as azure_eventhub_aio
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import config
+from ddtrace.contrib.internal.trace_utils import unwrap as _u
+from ddtrace.ext import azure_eventhubs as azure_eventhubsx
+from ddtrace.internal.schema import schematize_service_name
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.utils.formats import asbool
+from ddtrace.trace import Pin
+
+from .utils import create_context
+from .utils import dispatch_message_modifier
+
+
+config._add(
+    "azure_eventhubs",
+    dict(
+        _default_service=schematize_service_name("azure_eventhubs"),
+        distributed_tracing=asbool(os.getenv("DD_AZURE_EVENTHUBS_DISTRIBUTED_TRACING", default=True)),
+        batch_links=asbool(os.getenv("DD_TRACE_AZURE_EVENTHUBS_BATCH_LINKS_ENABLED", default=True)),
```

Comment:
Instead of `os.getenv`, can this be switched over to `_get_config` from https://github.com/DataDog/dd-trace-py/blob/55f2e27e29c150a1014717a5e2069f2411a6954f/ddtrace/internal/telemetry/__init__.py#L28?

A recent example of this being done is in this PR: https://github.com/DataDog/dd-trace-py/pull/14468/files/dd6d08f3164a3cccaa587bde4264991469728a72

Reply:
Done! See https://github.com/DataDog/dd-trace-py/pull/14636/commits/630c8301dbc11bd7927cd9d56d048e587f4e0f6e